common/src/main/java/com/wynntils/models/beacons/type/BeaconColor.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.beacons.type;
+
+import java.util.List;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import org.apache.commons.compress.utils.Lists;
+
+public enum BeaconColor {
+    GREEN((int) (0.78125 * 32), Items.GOLDEN_SHOVEL, ContentType.BOTH),
```

Comment:
Why these 0.xxx * 32 instead of normal integers?

Reply:
They are damage values taken from wynns resource pack

Reply:
And there they are represented as floats times 32? 

Even if they are, can't we just replace them with the normal integer damage value? To me, a damage value of `3` makes sense, while `0.09375 * 32` does not.

Reply:
I wanted to preserve how the resource pack has it which is 0-1 damage * itemDurability. But if you want to, we can make these integers